integration/external/Zerodev.test.ts

```diff
@@ -0,0 +1,90 @@
+import { HDNodeWallet, Wallet } from 'ethers'
+import { assert } from 'chai'
+import { ZeroDevEthersProvider, convertEthersSignerToAccountSigner } from '@zerodev/sdk'
+import {
+  Account,
+  AssetAttributes,
+  AssetPrice,
+  EthSignJWT,
+  Nevermined,
+  NeverminedOptions,
+} from '../../src'
+import { getMetadata } from '../utils'
+
+describe('Nevermined sdk with zerodev', () => {
+  let projectId: string
+  let owner: HDNodeWallet
+  let config: NeverminedOptions
+  let nevermined: Nevermined
+  let account: Account
+  let zerodevProvider: ZeroDevEthersProvider<'ECDSA'>
+
+  before(async () => {
+    projectId = process.env.PROJECT_ID!
+    owner = Wallet.createRandom()
+    account = new Account(await owner.getAddress())
+    const infuraToken = process.env.INFURA_TOKEN!
+
+    config = {
```

Comment:
Probably this can come from a `config.ts` file like the one we have for the regular integration tests